package.json

```diff
@@ -168,6 +169,7 @@
     "react-native-email-link": "^1.14.5",
     "react-native-fast-image": "^8.5.11",
     "react-native-fast-shadow": "^0.1.0",
+    "react-native-file-access": "^3.0.5",
```

Comment:
This seems unused?

Reply:
It was asked to be added by bugsnag

https://docs.bugsnag.com/performance/integration-guides/react-native/

It appears to be imported extensively in their code although it's not a peer dependency.